### ISSUE TITLE
Rule Proposal: No duplicate re-exports (no-duplicate-reexport)

### ISSUE THREAD

**Issue by SpainTrain** [benmosher/eslint-plugin-import#773]
With such a rule, the following would be considered a problem

```js
export {a} from './source-file'
export {b} from './source-file'
export {c} from './source-file'
```

The following would not be considered a problem

```js
export {a, b, c} from './source-file'
```

**Comment by ljharb** [comment]
It kind of seems like this should be an option on [no-duplicates](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-duplicates.md) instead of a separate rule.

**Comment by SpainTrain** [comment]
I tend to agree.  In fact, I had falsely assumed that `no-duplicates` enforced this.

**Comment by ljharb** [comment]
I'd say we'd want to add an "imports" option that defaults to `true`, and an "exports" option that defaults to `false`, and in the next semver-major, we can flip the default of "exports" to `true`.

**Comment by kevinwolfcr** [comment]
@ljharb  Sorry for bringing this up after all these years, but, is this still doable?

**Comment by ljharb** [comment]
Yep, just need a PR.
